experimental/form_app/lib/src/sign_in_http.g.dart

```diff
@@ -0,0 +1,19 @@
+// GENERATED CODE - DO NOT MODIFY BY HAND
+
+part of 'sign_in_http.dart';
+
+// **************************************************************************
+// JsonSerializableGenerator
+// **************************************************************************
+
+FormData _$FormDataFromJson(Map<String, dynamic> json) {
```

Comment:
Entirely your call, but consider rolling these by hand in order to remove the dependency on source_gen.